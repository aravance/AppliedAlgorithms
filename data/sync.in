3 30 25 35
6 25 25 25 25 25 25
2 15 30
8 20 21 30 23 29 25 27 22
2 19 20
10 10 11 12 13 14 86 87 88 89 90
9 10 20 30 40 50 60 70 80 90
2 10 12
2 10 14
2 10 26
2 10 25
10 90 90 90 90 90 90 90 90 90 90
10 10 10 10 10 10 10 10 10 10 10
7 10 11 13 15 21 32 38
10 10 11 12 13 14 15 16 17 18 19
9 10 11 12 15 32 37 38 39 42
10 10 11 12 15 20 32 37 38 39 42
0
