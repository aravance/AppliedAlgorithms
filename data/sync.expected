00:05:00
00:00:50
00:01:00
01:06:00
00:00:40
03:58:00
00:24:00
00:00:24
00:01:00
00:00:20
00:01:00
00:03:00
00:00:20
17:00:04
20:06:00
23:58:04
23:58:04
